recipes/gstools-cython/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "gstools-cython" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/gstools_cython-{{ version }}.tar.gz
+  sha256: 20ce39e21b0c1d30cedb9867c662655de1a123b7fcc7a52b4404efdc9736596d
+
+build:
+  skip: true  # [python_impl == 'pypy']
+  number: 0
+  script_env:
+    - GSTOOLS_BUILD_PARALLEL=1
+  script:
+    - {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython >=3.0                           # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - llvm-openmp  # [osx]
+    - libgomp      # [linux]
+    - {{ compiler('c') }}
+    - {{ stdlib("c") }}
+    - {{ compiler('cxx') }}
+  host:
+    - pip
+    - python
+    - setuptools >=64
+    - setuptools_scm >=7
+    - cython >=3.0
+    - numpy
+    - extension-helpers
+  run:
+    - python
+    - numpy >=1.20.0
+
+test:
+  imports:
+    - gstools_cython
+  requires:
+    - pytest
+  source_files:
+    - tests
+  commands:
+    - pytest -vv tests
+
+about:
+  home: https://github.com/GeoStat-Framework/GSTools-Cython
```

Comment:
This one points to a fork. Is that the canonical source? If not, can we fix it?

Reply:
It is the canonical source. Since we cut out the cython part of GSTools, I wanted to keep the history of these files and forked the GSTools repo.